designer/client/src/components/graph/node-modal/editors/map/Map.tsx

```diff
@@ -8,6 +8,7 @@ import MapKey from "./MapKey";
 import MapValue from "./MapValue";
 import { isEqual } from "lodash";
 import { useDiffMark } from "../../PathsToMark";
+import { DndItems } from "../../fragment-input-definition/DndItems";
```

Comment:
Maybe this should be moved to some `common` directory as it is no longer "fragment-specific" code?